third_party/silabs/SiWx917_sdk.gni

```diff
@@ -21,42 +21,29 @@ import("${chip_root}/src/lib/lib.gni")
 import("silabs_board.gni")
 
 declare_args() {
-  # Location of the efr32 SDK.
-  efr32_sdk_root = "${chip_root}/third_party/silabs/gecko_sdk"
   sdk_support_root = "${chip_root}/third_party/silabs/matter_support"
   wisemcu_sdk_root = "${chip_root}/third_party/silabs/wiseconnect-wifi-bt-sdk"
   wisemcu_sdk_root = "${chip_root}/third_party/silabs/wisemcu-wifi-bt-sdk"
   examples_plat_dir = "${chip_root}/examples/platform/silabs/SiWx917"
 
-  # Build openthread with prebuilt silabs lib
-  use_silabs_thread_lib = false
-  enable_openthread_cli = true
 
   kvs_max_entries = 255
 
   # Use Silabs factory data provider example.
   # Users can implement their own.
   use_efr32_factory_data_provider = true
 
-  # Enable Segger System View
-  use_system_view = false
```

Comment:
why is this removed? its a valuable tool to be able to use

Reply:
Yes it is a valuable tool but currently we are not using it.
And it is using files provided by Gecko SDK to build.

So once we have system view provided in Wisemcu SDK we will add it.